sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastExchangeExec.scala

```diff
@@ -514,33 +514,8 @@ abstract class GpuBroadcastExchangeExecBase(
 }
 
 object GpuBroadcastExchangeExecBase {
-  /**
-   * Create a thread factory that names threads with a prefix and also sets the threads to daemon.
-   */
-  private def namedThreadFactory(prefix: String): ThreadFactory = {
-    new ThreadFactoryBuilder().setDaemon(true).setNameFormat(prefix + "-%d").build()
-  }
-
-  /**
-   * Create a cached thread pool whose max number of threads is `maxThreadNumber`. Thread names
-   * are formatted as prefix-ID, where ID is a unique, sequentially assigned integer.
-   */
-  private def newDaemonCachedThreadPool(
-      prefix: String, maxThreadNumber: Int, keepAliveSeconds: Int = 60): ThreadPoolExecutor = {
-    val threadFactory = namedThreadFactory(prefix)
-    val threadPool = new ThreadPoolExecutor(
-      maxThreadNumber, // corePoolSize: the max number of threads to create before queuing the tasks
-      maxThreadNumber, // maximumPoolSize: because we use LinkedBlockingDeque, this one is not used
-      keepAliveSeconds,
-      TimeUnit.SECONDS,
-      new LinkedBlockingQueue[Runnable],
-      threadFactory)
-    threadPool.allowCoreThreadTimeOut(true)
-    threadPool
-  }
-
   val executionContext = ExecutionContext.fromExecutorService(
-    newDaemonCachedThreadPool("gpu-broadcast-exchange",
+    TrampolineUtil.newDaemonCachedThreadPool("gpu-broadcast-exchange",
```

Comment:
Don't need TrampolineUtil, this is already in a package under org.apache.spark.
```suggestion
    ThreadUtils.newDaemonCachedThreadPool("gpu-broadcast-exchange",
```